src/app.js

```diff
@@ -8,6 +8,235 @@ import './css/style.css';
 
 console.log('it loaded!');
 
+import Trip from './app/models/trip'
+
+import TripList from './app/collections/trip_list';
+
+import Reservation from './app/models/reservation'
+
+
+const TEXT_SEARCH = ['name', 'continent', 'category'];
+const TRIP_FIELDS = ['name', 'about', 'continent', 'category', 'weeks', 'cost'];
+const RESERVATION_FIELDS = ['tripId', 'name', 'email', 'age'];
+
+const trips = new TripList();
+
+let tripTemplate;
+
+let showTripTemplate;
+
+// Clear status messages
+const clearStatus = function clearStatus() {
+  $('#status-messages ul').html('');
+  $('#status-messages').hide();
+};
+
+// Add a new status message
+const reportStatus = function reportStatus(status, message) {
+  // console.log(`Reporting ${ status } status: ${ message }`);
+
+  // TODO: use an underscore method for this
+  // Should probably use an Underscore template here.
+
+  const statusHTML = `<li class="${ status }">${ message }</li>`;
+
+  // note the symetry with clearStatus()
+  $('#status-messages ul').append(statusHTML);
+  $('#status-messages').show();
+};
+
+// function for saving model
+const saveModel = function saveModel(modelToAdd, collection = 'none') {
+  modelToAdd.save({}, {
+    success: (model, response) => {
+      console.log(`Successfully saved!`);
+      if(collection != 'none') {
+        collection.add(model);
+      }
+      reportStatus('success', `Successfully saved!`);
```

Comment:
It would be good to clear old messages when you present a new group.  It gets rather large.